src/dep_logic/tags/platform.py

```diff
@@ -106,18 +106,21 @@ def current(cls) -> Self:
 
         platform_ = sysconfig.get_platform()
         platform_info = platform_.split("-", 1)
+        architecture: Optional[str] = None
         if len(platform_info) == 1:
             if platform_info[0] == "win32":
                 return cls(os.Windows(), Arch.X86)
             operating_system, _, version_arch = (
                 platform_.replace(".", "_").replace(" ", "_").partition("_")
             )
+            if version_arch.startswith("x86_64"):
```

Comment:
I feel it would better to move this to L126, then you don't need to modify L123

Reply:
Implemented. In the new position, it's more obvious that `startswith("x86_64")` is a catch-all that need not care about what follows. It's a good extension point if other esoteric platforms need to be supported :).